src/components/DetailContainer/types.ts

```diff
@@ -1,7 +0,0 @@
-import { ReactNode } from 'react';
```

Comment:
This is the better approach, keep types with each component, less files to lint, faster builds. As less files we have, the better.

Reply:
I am not sure if I understood you ... did you like my changes?

Reply:
Yes, my point is, having less file, speed up the whole build. Less I/O, so, cool to see less file 😎